qemu/tests/cfg/virtio_fsd_check_info.cfg

```diff
@@ -1,5 +1,6 @@
 - virtio_fsd_check_info:
     no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8.u0 Host_RHEL.m8.u1
```

Comment:
It's better to remove `Host_RHEL.m6 Host_RHEL.m7` here.

Reply:
ok, but could you please explain here?

Reply:
Sure.
Since we already limit `only Host_RHEL.m8`, we do not need `no Host_RHEL.m6 Host_RHEL.m7`.

Reply:
you are right, that seems a issue, I'll update my code

Reply:
updated